### PR TITLE
[doc] add tpc-h benchmark

### PR DESCRIPTION
**Pull request by hf200012**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/benchmark/tpch.md [hunk 1]
```diff
@@ -0,0 +1,973 @@
...
+
+## 3. Test Data Volume
+
+The entire test simulation generates 100G of data and is imported into Doris 0.15.0 RC04 and Doris 1.1 versions for testing. The following is the relevant description of the table and the amount of data.
+
+| TPC-H Table Name | Rows   | data size  | remark |
+| :--------------- | :----- | ---------- | :----- |
+| REGION           | 5      | 400KB      |        |
+| NATION           | 25     | 7.714 KB   |        |
+| SUPPLIER         | 100万  | 85.528 MB  |        |
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:66]: Chinese in english doc

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 2]
```diff
@@ -0,0 +1,974 @@
...
+
+执行上面的测试 SQL 或者 执行下面的命令
+
+```
+./run-tpch-queries.sh
+```
+
+>注意：
+>
+>1. 目前Doris的查询优化器和统计信息功能还不完善，所以我们在TPC-H中重写了一些查询以适应Doris的执行框架，但不影响结果的正确性
```
Comment (morningman) [inline docs/zh-CN/docs/benchmark/tpch.md:968]: 建议吧这块放到开头的 “注意” 中。

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 3]
```diff
@@ -0,0 +1,974 @@
...
+| SUPPLIER  | 100万  | 85.528 MB  | 供应商表     |
+| PART      | 2000万 | 752.330 MB | 零部件表     |
+| PARTSUPP  | 8000万 | 4.375 GB   | 零部件供应表 |
+| CUSTOMER  | 1500万 | 1.317 GB   | 客户表       |
+| ORDERS    | 1.5亿  | 6.301 GB   | 订单表       |
+| LINEITEM  | 6亿    | 20.882 GB  | 订单明细表   |
+
+## 4. 测试SQL
+
+```sql
```
Comment (morningman) [inline docs/zh-CN/docs/benchmark/tpch.md:75]: 建议这里之间指向代码库的 query 目录连接，否则后续有修改，我们还得同时修改文档和代码。

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 4]
```diff
@@ -0,0 +1,974 @@
...
+>
+> 注2：本文档涉及的操作都在 CentOS 7.x 上进行测试。
+
+在 TPC-H 标准测试数据集上的 22 个查询上，我们对即将发布的 Doris 1.1 版本和 Doris 0.15.0 RC04 版本进行了对别测试，整体性能提升了 3-4 倍。个别场景下达到十几倍的提升。
+
+![image-20220614114351241](/images/image-20220614114351241.png)
+
+## 1. 硬件环境
+
+| 机器数量 | 4 台腾讯云主机（1个FE，3个BE）       |
```
Comment (morningman) [inline docs/zh-CN/docs/benchmark/tpch.md:43]: 修改下header

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 5]
```diff
@@ -0,0 +1,974 @@
...
+#### 6.3.2 执行以下脚本生成创建 TPC-H 表
+
+```shell
+sh create-tpch-tables.sh
+```
+或者复制 [create-tpch-tables.sql](https://github.com/apache/incubator-doris/blob/master/tools/tpch-tools/create-tpch-tables.sql) 中的建表语句，在 Doris 中执行。
+
+下面是建表语句。
+
+```sql
```
Comment (morningman) [inline docs/zh-CN/docs/benchmark/tpch.md:771]: 这里去掉吧，保留连接就行了

**Comment by hf200012** [comment]
OK，solved

File: docs/en/docs/benchmark/tpch.md [hunk 6]
```diff
@@ -59,16 +59,16 @@ On 22 queries on the TPC-H standard test dataset, we tested the upcoming Doris 1
...
+| TPC-H Table Name | Rows        | data size  | remark |
+| :--------------- | :---------- | ---------- | :----- |
+| REGION           | 5           | 400KB      |        |
+| NATION           | 25          | 7.714 KB   |        |
+| SUPPLIER         | 100 million | 85.528 MB  |        |
+| PART             | 20 million  | 752.330 MB |        |
+| PARTSUPP         | 80 million  | 4.375 GB   |        |
+| CUSTOMER         | 15 million  | 1.317 GB   |        |
+| ORDERS           | 1.5 billion | 6.301 GB   |        |
+| LINEITEM         | 6 billion   | 20.882 GB  |        |
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:72]:
```suggestion
| SUPPLIER         | 1 million | 85.528 MB  |        |
| PART             | 20 million  | 752.330 MB |        |
| PARTSUPP         | 80 million  | 4.375 GB   |        |
| CUSTOMER         | 15 million  | 1.317 GB   |        |
| ORDERS           | 150 million | 6.301 GB   |        |
| LINEITEM         | 600 million   | 20.882 GB  |        |
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+sh gen-tpch-data.sh
+```
+
+> 注1：通过 `sh gen-tpch-data.sh -h` 查看脚本帮助。
+>
+> 注2：数据会以 `.tbl` 为后缀生成在  `tpch-data/` 目录下。文件总大小约100GB。生成时间可能在数分钟到1小时不等。
+>
+> 注3：默认生成 100G 的标准测试数据集
+
+### 6.3 建表
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:159]:
```suggestion
### 7.3 建表
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+
+#### 6.3.1 准备 `doris-cluster.conf` 文件
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:161]:
```suggestion
#### 7.3.1 准备 `doris-cluster.conf` 文件
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+export FE_QUERY_PORT=9030
+# Doris username
+export USER='root'
+# Doris password
+export PASSWORD=''
+# The database where TPC-H tables located
+export DB='tpch1'
+```
+
+#### 6.3.2 执行以下脚本生成创建 TPC-H 表
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:184]:
```suggestion
#### 7.3.2 执行以下脚本生成创建 TPC-H 表
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 10]
```diff
@@ -0,0 +1,232 @@
...
+
+```shell
+sh create-tpch-tables.sh
+```
+或者复制 [create-tpch-tables.sql](https://github.com/apache/incubator-doris/blob/master/tools/tpch-tools/create-tpch-tables.sql) 中的建表语句，在 Doris 中执行。
+
+
+### 6.4 导入数据
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:192]:
```suggestion
### 7.4 导入数据
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+
+通过下面的命令执行数据导入：
+
+```shell
+sh ./load-tpch-data.sh
+```
+
+### 6.5 检查导入数据
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:200]:
```suggestion
### 7.5 检查导入数据
```

File: docs/zh-CN/docs/benchmark/tpch.md [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+select count(*)  from  partsupp;
+select count(*)  from  part;
+select count(*)  from  customer;
+select count(*)  from  supplier;
+select count(*)  from  nation;
+select count(*)  from  region;
+select count(*)  from  revenue0;
+```
+
+### 6.6 查询测试
```
Comment (morrySnow) [inline docs/zh-CN/docs/benchmark/tpch.md:216]:
```suggestion
### 7.6 查询测试
```

File: docs/en/docs/benchmark/tpch.md [hunk 13]
```diff
@@ -0,0 +1,229 @@
...
+sh gen-tpch-data.sh
+```
+
+> Note 1: View script help via `sh gen-tpch-data.sh -h`.
+>
+> Note 2: The data will be generated in the `tpch-data/` directory with the suffix `.tbl`. The total file size is about 100GB. The generation time may vary from a few minutes to an hour.
+>
+> Note 3: The standard test data set of 100G is generated by default
+
+### 6.3 Create Table
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:156]:
```suggestion
### 7.3 Create Table
```

File: docs/en/docs/benchmark/tpch.md [hunk 14]
```diff
@@ -0,0 +1,229 @@
...
+export FE_QUERY_PORT=9030
+# Doris username
+export USER='root'
+# Doris password
+export PASSWORD=''
+# The database where TPC-H tables located
+export DB='tpch1'
+```
+
+#### 6.3.2 Execute the following script to generate and create the TPC-H table
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:181]:
```suggestion
#### 7.3.2 Execute the following script to generate and create the TPC-H table
```

File: docs/en/docs/benchmark/tpch.md [hunk 15]
```diff
@@ -0,0 +1,229 @@
...
+
+> Note 1: View script help via `sh gen-tpch-data.sh -h`.
+>
+> Note 2: The data will be generated in the `tpch-data/` directory with the suffix `.tbl`. The total file size is about 100GB. The generation time may vary from a few minutes to an hour.
+>
+> Note 3: The standard test data set of 100G is generated by default
+
+### 6.3 Create Table
+
+#### 6.3.1 Prepare the `doris-cluster.conf` file
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:158]:
```suggestion
#### 7.3.1 Prepare the `doris-cluster.conf` file
```

File: docs/en/docs/benchmark/tpch.md [hunk 16]
```diff
@@ -0,0 +1,229 @@
...
+
+### 6.4 导入数据
+
+通过下面的命令执行数据导入：
+
+```shell
+sh ./load-tpch-data.sh
+```
+
+### 6.5 Check Imported Data
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:197]:
```suggestion
### 7.5 Check Imported Data
```

File: docs/en/docs/benchmark/tpch.md [hunk 17]
```diff
@@ -0,0 +1,229 @@
...
+select count(*)  from  partsupp;
+select count(*)  from  part;
+select count(*)  from  customer;
+select count(*)  from  supplier;
+select count(*)  from  nation;
+select count(*)  from  region;
+select count(*)  from  revenue0;
+```
+
+### 6.6 Query Test
```
Comment (morrySnow) [inline docs/en/docs/benchmark/tpch.md:213]:
```suggestion
### 7.6 Query Test
```

**Pull request closed without merging**